example-lint

```diff
@@ -27,7 +28,8 @@ def run() -> None:
     ]  # type: List[str]
     by_lang = linter_config.list_files(file_types, exclude=EXCLUDED_FILES)
 
-    linter_config.external_linter('mypy', ['tools/run-mypy'], ['py'], pass_targets=False,
+    mypy_command = ['python', 'tools\\run-mypy'] if sys.platform == 'win32' else ['tools/run-mypy']
```

Comment:
Shouldn’t `[sys.executable, 'tools/run-mypy']` do the right thing on both platforms?